src/__tests__/json-schema.test.js

```diff
@@ -0,0 +1,314 @@
+describe('json-schema', () => {
+  const Sequelize = require('sequelize')
+  const sequelize = new Sequelize('postgres://fakeUser:pass@fake.com:1111/fake')
+  const SEQUELIZE_JSON_TABLE = [
+    [
+      'string',
+      Sequelize.STRING,
+      { type: 'string' }
+    ],
+    [
+      'string',
+      Sequelize.CHAR,
+      { type: 'string' }
+    ],
+    [
+      'string 123',
+      Sequelize.STRING(1234),
+      {
+        type: 'string',
+        maxLength: 1234
+      }
+    ],
+    [
+      'text',
+      Sequelize.TEXT,
+      { type: 'string' }
+    ],
+    [
+      'uuid',
+      Sequelize.UUID,
+      {
+        type: 'string',
+        format: 'uuid'
+      }
+    ],
+    [
+      'integer',
+      Sequelize.INTEGER,
+      { type: 'integer' }
+    ],
+    [
+      'integer',
+      Sequelize.BIGINT,
+      { type: 'integer' }
+    ],
+    [
+      'float',
+      Sequelize.FLOAT,
+      { type: 'number' }
+    ],
+    [
+      'float 11',
+      Sequelize.FLOAT(11),
+      {
+        type: 'number',
+        maximum: 11
```

Comment:
Same question here as for string length.

Reply:
json's maximum is the inclusive maximum number.

sequelize's floating type (Float, Real, Double...etc) length refers to the number of digits excluding the decimal. E.g. DOUBLE(8, 3) where 8 is the total no. of digits excluding the decimal point, and 3 is the no. of digits to follow the decimal. https://code.rohitink.com/2013/06/12/mysql-integer-float-decimal-data-types-differences/